.github/workflows/delegation-pop-verify.yml

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2023 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Verify POP for a delegation
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [main]
+    paths:
+      - 'repository/**'
+  pull_request:
+    branches: [main]
+
+jobs:
+  verify:
+    if: startsWith(github.event.pull_request.title, 'feat: add-delegtion for ')
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Veriry POP
```

Comment:
you may need to add a step here or inside the wrapper to build the `tuf` binary (depending on the docs of how you want to write user verification: if local users call the wrapper directly, then add the step in the wrapper, or if you want to instruct verifiers to run `./scripts/step-0.sh` first that builds it as well).

Reply:
Ah, yes. Thanks. The workflow file now is more like an idea than something that I expect to work, just made it so I can think around a model that ties all things together.